spec/support/schemas/claims_api/v2/veterans/claims/claim.json

```diff
@@ -132,6 +132,45 @@
       "nullable": true,
       "description": "If true, indicates a decision has been requested and/or a Waiver 5103 has been submitted",
       "example": "false"
+    },
+    "trackedItems": {
```

Comment:
Can we move `trackedItems` up above `5103WaiverSubmitted` so this file and the generated dev docs will maintain their alphabetization?